Sources/SwiftFortuneWheel/Utils/Audio/AudioPlayable.swift

```diff
@@ -47,4 +47,9 @@ extension AudioPlayable {
         guard let _audioFile = audioFile else { return }
         playSound(audioFile: _audioFile)
     }
+    
+    /// Pause sound(s)
+    func pauseAllSound() {
+        audioPlayerManager.pauseAll()
```

Comment:
Since sound playback is optional, it should be guarded first, check if the sound is playable or not before calling the pause method. rename the method so that it is immediately clear that it is optional.

Reply:
Hmm. I guess I got your point. Let's revisit this when I've spare time.